pkg/manager/member/pvc_resizer.go

```diff
@@ -111,6 +112,21 @@ func (p *pvcResizer) Resize(tc *v1alpha1.TidbCluster) error {
 			return err
 		}
 	}
+	// patch TiDB PVCs
+	if tc.Spec.TiDB != nil {
+		tidbMemberType := v1alpha1.TiDBMemberType.String()
+		for _, sv := range tc.Spec.TiDB.StorageVolumes {
+			key := fmt.Sprintf("%s-%s-%s-%s", tidbMemberType, sv.Name, tc.Name, tidbMemberType)
+			if quantity, err := resource.ParseQuantity(sv.StorageSize); err == nil {
+				pvcPrefix2Quantity[key] = quantity
```

Comment:
Should we define dedicated `pvcPrefix2Quantity` for different components?

Reply:
Good one.
Fixed.